src/cli/run.js

```diff
@@ -118,7 +129,7 @@ console.log(Object.keys(resolved).length, 'resolved queries');
 
 /** Step (4) */
 
-const schemaCache = {};
+const schemaCache: {[key: string]: [GraphQLSchema, Schema]} = {};
```

Comment:
FYI, one of our TS [best practices](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2201682700/TypeScript+Best+Practices) is to use `Record<K, V>` instead of `{[key: K]: V}`.  There's a lint rule for this that I'll be rolling out later so no changes req'd in this PR.